tests/app/questionnaire/test_path_finder.py

```diff
@@ -704,3 +704,29 @@ def test_full_routing_path_with_repeating_sections(self):
         ]
 
         self.assertEqual(routing_path, expected_path)
+
+    def test_enabled_section_ids(self):
+        schema = load_schema_from_name('test_section_enabled_checkbox')
+        progress_store = ProgressStore(
+            [
+                {
+                    'section_id': 'section-1',
+                    'block_ids': ['section-1-block'],
+                    'status': 'COMPLETED',
+                }
+            ]
+        )
+
+        answer_store = AnswerStore(
+            [{'answer_id': 'section-1-answer', 'value': ['Section 2']}]
+        )
+        path_finder = PathFinder(
+            schema=schema,
+            answer_store=answer_store,
+            metadata={},
+            progress_store=progress_store,
+        )
+
+        expected_section_ids = ['section-1', 'section-2', 'summary-section']
+
+        self.assertEqual(path_finder.enabled_section_ids, expected_section_ids)
```

Comment:
It might also be worth asserting that `section-3` isn't in `enabled_section_ids`.

Reply:
Can do but the current assert makes asserting `section-3` isn't in `enabled_section_ids` redundant?.

Reply:
Oh yeah 🤦‍♂